.cargo/config.offline

```diff
@@ -9,7 +9,7 @@ git = "https://github.com/zcash/librustzcash.git"
 rev = "d14e7a707ce01cefcbc82651dad48f002185dded"
 replace-with = "vendored-sources"
 
-[source."https://github.com/nuttycom/hdwallet.git"]
+[source."https://github.com/nuttycom/hdwallet"]
 git = "https://github.com/nuttycom/hdwallet.git"
```

Comment:
I'd prefer that these stay in sync, so my preferred resolution is actually to not make this change here, but instead add a `.git` suffix to the corresponding patch in `Cargo.toml`.